src/extensions/agent_modules/cloud_communication/iothub_client/adu_core_interface/src/startup_msg_helper.c

```diff
@@ -112,10 +112,7 @@ bool StartupMsg_AddCompatPropertyNames(JSON_Object* startupObj)
     bool success = false;
 
     JSON_Status jsonStatus = json_object_set_string(
-        startupObj,
-        ADUCITF_FIELDNAME_COMPAT_PROPERTY_NAMES,
-        IsNullOrEmpty(config->compatPropertyNames) ? DEFAULT_COMPAT_PROPERTY_NAMES_VALUE
-                                                   : config->compatPropertyNames);
```

Comment:
Do we still need this function?

Reply:
no but will remove along when remove cloud_communication/iothub_client